qdrant-landing/content/documentation/concepts/collections.md

```diff
@@ -101,6 +107,10 @@ client.createCollectionAsync("test_collection",
         VectorParams.newBuilder().setDistance(Distance.Dot).setSize(4).build()).get();
 ```
 
+If successful, these commands write data to the `qdrant_storage` subdirectory. You
+can verify changes in the `config.json` file in the directory associated with
```

Comment:
Do you think we need to expose internal structure in docs?

Reply:
We don't have to. IMO the responses to these code samples should be "good enough". (My work heritage is heavy on open source, so my instinct is to 'expose everything'.) I'll delete these lines.